indico/client/client.py

```diff
@@ -64,3 +64,16 @@ def call(self, request: Union[HTTPRequest, RequestChain]):
             return self._handle_request_chain(request)
         elif request and isinstance(request, HTTPRequest):
             return self._http.execute_request(request)
+
+    def paginate(self, request: PagedRequest):
```

Comment:
if we're cool rolling with this - any hesitation with also including the change from the same PR to paginate ListSubmissions?